### ISSUE TITLE
Definition for rule 'import/no-unresolved' was not found.

### ISSUE THREAD

**Issue by vaynevayne** [import-js/eslint-plugin-import#2793]
```jsx
// .eslint.rc
{
  "env": {
    "browser": true,
    "es2021": true
  },
  "extends": [
    "eslint:recommended",
    "plugin:react/recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:react-hooks/recommended",
    "plugin:import/typescript",
    "prettier"
  ],
  "settings": {
    "import/resolver": {
      "typescript": true,
      "node": true
    }
  },
  "overrides": [],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": "latest",
    "sourceType": "module"
  },
  "plugins": [
    "react",
    "react-refresh",
    "@typescript-eslint",
    "@tanstack/query"
  ],
  "rules": {
    "react/react-in-jsx-scope": "off",
    "react/self-closing-comp": ["error"],
    "quotes": ["error", "single"],
    "semi": ["error", "never"],
    "react/display-name": ["off"],
    "@typescript-eslint/no-explicit-any": "off",
    "@tanstack/query/exhaustive-deps": "error",
    "react-refresh/only-export-components": "warn",
    "import/no-unresolved": "error"
  }
}

```

error 
```jsx
// context.ts
import { createContext } from 'react'
import { ReportConfig } from './type'

export const StoreContext = createContext<ReportConfig>(null)


```

**Comment by ljharb** [comment]
You're missing `import` in your plugins array.

**Comment by vaynevayne** [comment]
`"plugin:import/typescript"`  Isn't the plugin included in this package?

**Comment by ljharb** [comment]
nope: https://github.com/import-js/eslint-plugin-import/blob/main/config/typescript.js
